dicoogle/src/main/resources/webapp/js/components/search/result/patientView.js

```diff
@@ -138,7 +135,7 @@ const PatientView = createReactClass({
         <Checkbox
           label=""
           onChange={this.handleSelect.bind(this, item)}
-          ref={this.handleRefs.bind(this, id)}
+          inputRef={ref => this.refsClone[id] = ref}
```

Comment:
This `handleRefs` as a separate function was intentional, so that it did not have to create an arrow function on each render call (more info [here](https://reactjs.org/docs/handling-events.html)). If `ref` instead of `inputRef` was the real problem, then `handleRefs` could be brought back.

Reply:
Yes, I agree. There is one more mistake at patient level, going to fix it.